assured-workloads-example/assured_workload_mgmt.tf

```diff
@@ -30,3 +30,18 @@ module "aw_mgmt_project" {
     "cloudresourcemanager.googleapis.com"
   ]
 }
+
+module "aw_workload_project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 16.0"
+
+  project_id                  = "${var.aw_base_id}-workload-${local.default_suffix}"
+  disable_services_on_destroy = true
+  org_id                      = var.organization_id
+  folder_id                   = var.aw_root_folder_id
```

Comment:
Shouldn't the folder here be `aw_consumer_folder_id` instead of `aw_root_folder_id`?

If I understood correctly, `aw_root_folder_id` folder itself is outside Assured Workload scope.

Reply:
If the statement I made above is correct, we might want to rename the variables because it can mislead the user.
`aw_root_folder_id` is not the AW root folder but the parent folder of AW instead, while `aw_consumer_folder_id` is the actual AW root folder.

Reply:
Yes, the management project should be out of Assured Workloads folder.
Yes, let's discuss better names. There's another review that may impact the way this project is created, so I'm going to wait the verdict there before changing here. Thanks for the review.